# HSS Koordinat bilgileri
int32 id
float64 enlem
float64 boylam
int32 yaricap
bool is_active
